main_test.go

```diff
@@ -104,3 +106,71 @@ func SortByStringValueOnKey(field string, data []interface{}) {
 func ErrorHandlingContext(handler echo.HandlerFunc) func(echo.Context) error {
 	return middleware.HandleErrors(handler)
 }
+
+// TestInvalidIdsReturn400 tests that when the handlers are given invalid IDs, these return a bad request response back
+// to the client.
+func TestInvalidIdsReturn400(t *testing.T) {
+
+	testData := []struct {
+		Method      string
+		IdParamName string
+		Handler     func(c echo.Context) error
```

Comment:
I placed it in `main_test.go` because to be honest I didn't know where to put it. However I don't think it belongs to `middleware/`, since I'm testing all the handlers, so that's why I thought it should go on the main package — also the second reason is that I cannot access the handlers from within a submodule.

Reply:
id_validation_handlers_test.go sounds good ⭐️

Reply:
Done! Thank you!